### ISSUE TITLE
Scoping JSX comment brackets

### ISSUE THREAD

**Issue by LeonardoGentile** [babel/babel-sublime#279]
It would be nice if jsx comment could be scoped as comment somehow:

```
{/* comment */}
```

The opening bracket is scoped as: 

```
source.js
meta.class.js
meta.group.braces.curly.js
meta.group.braces.round.js
meta.jsx.js
meta.embedded.expression.jsx
punctuation.section.embedded.begin.jsx
```

I am using a custom color for highlight the opening and closing brackets for JSX expressions  to make them more visible. 

But I would also like to render the two blocks `{/*` and `*/}` (and all their contents) with the same colors of all the other comments in js. This is not possible because the the block `{/*` `*/}` are not scoped. While the inner content `{/* CONTENT */}` is correctly scoped as comment.

Would it be possible to scope the opening and closing jsx comment brackets? Thansk

**Comment by zertosh** [comment]
Hmm, I'm not sure I understand what you're asking. Using this notation, what is it you'd like to see?

```
<div>{/* comment */}</div>
//^ meta.jsx meta.tag entity.name.tag
//<- meta.jsx meta.tag punctuation.definition.tag.begin
//  ^ meta.jsx meta.tag punctuation.definition.tag.end
//   ^ meta.jsx meta.embedded.expression punctuation.section.embedded.begin
//    ^^^^^^^^^^^^^ meta.jsx meta.embedded.expression comment.block
//    ^^ meta.jsx meta.embedded.expression comment.block punctuation.definition.comment.begin
//               ^^ meta.jsx meta.embedded.expression comment.block punctuation.definition.comment.end
//                 ^ meta.jsx meta.embedded.expression punctuation.section.embedded.end
//                  ^^ meta.jsx meta.tag punctuation.definition.tag.begin
//                    ^^^ meta.jsx meta.tag entity.name.tag
//                       ^ meta.jsx meta.tag punctuation.definition.tag.end
```

**Comment by LeonardoGentile** [comment]
I'd like the entire block `{/*`(including the bracket)  to be scoped as
```
source.js 
meta.jsx.js
meta.embedded.expression.jsx
comment.block.js
punctuation.definition.comment.begin.js
```
and the entire closing block `*/}` (including the bracket) to be scoped as 
```
source.js
meta.jsx.js
meta.embedded.expression.jsx
comment.block.js
punctuation.definition.comment.end.js
```

**Comment by zertosh** [comment]
Hmm, this may fix an issue with uncommenting. I'll circle back to this in #293.

**Comment by jcrben** [comment]
@zertosh do you mean this could fix the issue with the keyboard shortcut toggling comments discussed in https://github.com/babel/babel-sublime/issues/262?

**Comment by ukazap** [comment]
any news on this?

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
